egs2/dirha_wsj/asr1/conf/lm.yaml

```diff
@@ -0,0 +1,10 @@
+optim: sgd
+patience: 3
+max_length: 150 # 40 for word LMs
```

Comment:
I changed naming rule of config files to test in ci. so please follow #1617

Reply:
Changed the name to train_lm.yaml.